stability/stability/windows_native-stability-test.py

```diff
@@ -148,7 +148,7 @@ def runParsec():
     targetFile = os.path.join(targetDir, 'blackscholes.tar.gz')
     downloadAndUnpack(benchmarkLocation, targetFile)
     
-    return runAndProcess("%WORKSPACE%\\blackscholes_cpp_serial 1 in_10M.txt prices.txt", parsecProcessResults)
+    return runAndProcess("START \"STABILITY_PERF_RUN\" /B /WAIT /HIGH /AFFINITY 0x2 %WORKSPACE%\\blackscholes_cpp_serial 1 in_10M.txt prices.txt", parsecProcessResults)
```

Comment:
I think you need to prefix the command with `cmd.exe /c`. START is a cmd.exe command and might now be understood outside the cmd.exe process.